src/microbe_stage/CompoundBag.cs

```diff
@@ -38,13 +37,25 @@ public CompoundBag(float capacity)
     /// <summary>
     ///   Gets the capacity for a given compound
     /// </summary>
-    /// <returns>Returns <see cref="Capacity"/> if the compound is useful, otherwise 0</returns>
+    /// <returns>
+    ///   Returns compoundCapacities[compound] if the compound has a specific capacity set,
+    ///   or nominalCapacity if the compound is useful, otherwise 0
```

Comment:
```suggestion
    ///   Returns the capacity this bag has for storing the compound if it is useful, otherwise 0
```
I'd omit the internal details, especially the non `<see ...` type reference to a private variable.